packages/frontend/composables/.vscode/extensions.json

```diff
@@ -0,0 +1,3 @@
+{
+	"recommendations": ["Vue.volar"]
```

Comment:
Should we have these separately for different packages or only in the repo root?